resources/messages/chs.ini

```diff
@@ -0,0 +1,151 @@
+; chs.ini
```

Comment:
Why is this chs.ini? The ISO639-2 code for Chinese should be `chi` (bibliographic) or `zho` (terminology).

Reply:
chs means simplified Chinese and its traditional counterpart is cht. This is a generic language code. https://help.bing.microsoft.com/#apex/bing/en-us/10004/-1

Reply:
@SuperYYT it isn't. it's a second-level region code. `chs` alone doesn't make sense without `zh`.

Reply:
But is `chs` used in the client anyway?

Reply:
yeah >_<